src/features/pastelID/PastelID.tsx

```diff
@@ -25,6 +41,16 @@ function passphraseStatusColor(validation: TPasswordStrengthResult) {
   return colors[validation.id]
 }
 
+function createPastelKeysFolder() {
+  const rootPath = locatePastelConfDir()
+  if (rootPath) {
+    const pastelKeysFolder = path.join(rootPath, 'pastelkeys')
+    if (!fs.existsSync(pastelKeysFolder)) {
+      fs.mkdirSync(pastelKeysFolder)
```

Comment:
Please use unblocking promise methods: fs.promises.access and fs.promises.mkdir instead. Sync methods are blocking main (UI) thread.

Reply:
Fixed